src/actions/actions.types.ts

```diff
@@ -8,6 +8,7 @@ export type ActionType =
   | "copyToClipboard"
   | "cutToClipboard"
   | "deselect"
+  | "editNew"
```

Comment:
Added this so that paste and insert snippet can use it; could be useful to map a spoken form at some point 🤷.  Kinda like a fancy "change" that will insert delimiters if you give it a positional modifier

Reply:
Then maybe we should just merge this with change and allowed that option?

Reply:
That was my suggestion at meet-up, but the issue is that highlighting becomes a bit complex, as the targets will need to own a bit more. I'm not opposed to it, but maybe out of scope for this PR? We'll need to think a bit about giving targets more autonomy wrt highlighting

Reply:
I think we can do that in a follow up pr

Reply:
I mean we already have removal highlight range in the targets so we could just add content highlight range if we want.

Reply:
I think there is a fairly clean solution here that would also simplify the highlight graph component, but it's worth pairing on and I think out of scope

If you prefer for this PR I can just have the paste action keep its own `EditNew` object; that was my original solution